src/components/ItemCard/ItemCard.jsx

```diff
@@ -0,0 +1,14 @@
+import { Tag, TagLabel, TagCloseButton } from '@chakra-ui/react';
+import React from 'react';
+
+const ItemCard = ({ name }) => {
```

Comment:
🚫 **[eslint]** <[react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'name' is missing in props validation